mycroft/util/lang/parse_en.py

```diff
@@ -1039,15 +1040,21 @@ def normalize_en(text, remove_articles):
                          "you are not", "you are not", "you are", "you have"]
             word = expansion[contraction.index(word)]
 
-        # Convert numbers into digits, e.g. "two" -> "2"
-        textNumbers = ["zero", "one", "two", "three", "four", "five", "six",
-                       "seven", "eight", "nine", "ten", "eleven", "twelve",
-                       "thirteen", "fourteen", "fifteen", "sixteen",
-                       "seventeen", "eighteen", "nineteen", "twenty"]
-
-        if word in textNumbers:
-            word = str(textNumbers.index(word))
-
         normalized += " " + word
 
+    # needs to be here to avoid import error
+    from mycroft.util.parse import extract_numbers
```

Comment:
I'm not a fan of doing these kinds of workarounds for cyclic imports.

What do you think of creating a base method for extract_numbers in `lang/parse_common.py` which takes the extract_number function as an argument and a just a thin wrapper function in `util/parse.py`?

Reply:
sounds good!